.gitignore

```diff
@@ -36,14 +36,22 @@ reportJunit.xml
 
 !lib-injection/build/docker/ruby/lib_injection_rails_app/bin
 !utils/build/docker/ruby/rails52/bin
-/utils/build/docker/dotnet/.vs/app/*
-/utils/build/docker/dotnet/app.csproj.user
```

Comment:
[Note to reviewers] These are handled below with `.vs/` and `**/dotnet/**/*.user`.